README.md

```diff
@@ -79,7 +79,97 @@ java -javaagent:path/to/opentelemetry-javaagent-all.jar \
      -jar myapp.jar
 ```
 
-### Configuration parameters (subject to change!)
+### Manually instrumenting
```

Comment:
I'm not sure that the "Manually instrumenting" section should go above the "Configuration parameters" section?

Are you feeling like the README is getting too big, and stuff lower down is getting lost? Maybe we can split more things out into sub-pages and keep the main README lighter?

Reply:
Yes, that was the main issue I was trying to address here. If possible, I'd like to keep "restructuring" to another PR.

Reply:
> If possible, I'd like to keep "restructuring" to another PR.

No problem. I'm still not sure about "Manual instrumentation" section going to the top though. My thinking is that "Manual instrumentation" section is generally something users need after they first get up and running with auto-instrumentation.

Reply:
I've moved it back below and added a small sentence in the Getting Started section. Let me know if that works for you.